src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -67,10 +70,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
             throws IOException, ServletException {
         String username = (String) request.getSession().getAttribute("user");
         User user = userStore.getUser(username);
-
+        String status = request.getParameter("status");
         String bio = request.getParameter("bio");
-        user.setUserBio(bio);
-        userStore.updateUserData(user, "bio", bio);
+
+        if(status == null){
+            user.setUserBio(bio);
+            userStore.updateUserData(user, "bio", bio);}
```

Comment:
nit: make sure you format code according to the style guide. I don't want to be too strict about it but stuff like this sticks out :)